### DISCUSSION TITLE
Static website / Gatsby to create large blogs

### DISCUSSION THREAD

**Discussion by ghost** [strapi/strapi#6102] [Legacy Topics]
Hey !

I’m very interested by « Jamstack speed ».

Is it recommended to used jamstack to create blog with thousands « builds » per days ?

Can we limit builds each day in Strapi config’ ? For instance, 1 build per hour ? 

No need to be update each minute for my website...

Thanks

**Comment by robertpiosik** [comment]
In frequently updated database I would consider creating a custom webhook logic in [afterSave hook](https://strapi.io/documentation/3.0.0-beta.x/concepts/models.html#life-cycle-callbacks).

**Reply by ghost** [reply]
Thanks 👍🏻

**Reply by BramDecuypere** [reply]
Is there a way to do 'global' afterSave. So like on any update of the model, send a post to an outside hook? Or should we create a 'common' package and trigger that in every model?
